crates/dbsp/Cargo.toml

```diff
@@ -92,34 +88,29 @@ nix = { version = "0.27.1", features = ["uio", "feature", "fs"] }
 io-uring = "0.6.3"
 
 [dev-dependencies]
-rand = "0.8.5"
-proptest-derive = { version = "0.5.0" }
-proptest = { version = "1.5.0" }
-proptest-state-machine = { version = "0.3.0" }
-futures = { version = "0.3.30", features = ["executor"] }
-pretty_assertions = { version = "1.4" }
-csv = "1.2.2"
-tar = "0.4.38"
-zstd = "0.12.0"
-criterion = "0.5.1"
-pprof = { version = "0.13.0", features = ["flamegraph", "criterion"] }
-rand_xoshiro = "0.6.0"
-indicatif = "0.17.0-rc.11"
-#clap = { version = "3.2.8", features = ["derive", "env"] }
-reqwest = { version = "0.11.11", features = ["blocking"] }
-chrono = { version = "0.4.38", features = ["rkyv-64", "serde"] }
-env_logger = { version = "0.11.2", features = ["color"] }
-serde_json = "1.0.127"
+rand = { workspace = true }
+proptest-derive = { workspace = true }
+proptest = { workspace = true }
+proptest-state-machine = { workspace = true }
+futures = { workspace = true, features = ["executor"] }
+pretty_assertions = { workspace = true }
+csv = { workspace = true }
+tar = { workspace = true }
+zstd = { workspace = true }
+criterion = { workspace = true }
+pprof = { workspace = true, features = ["flamegraph", "criterion"] }
+rand_xoshiro = { workspace = true }
+indicatif = { workspace = true }
+reqwest = { workspace = true, features = ["blocking"] }
+chrono = { workspace = true, features = ["rkyv-64", "serde"] }
+env_logger = { workspace = true, features = ["color"] }
+serde_json = { workspace = true }
 ijson = { version = "0.1.4", package = "feldera-ijson" }
 
 [[bench]]
 name = "galen"
 harness = false
 
-#[[bench]]
```

Comment:
no consideration to reinstate these?

Reply:
they have been uncommented for a long time now, if someone wants them they can put them back